test/data/estimate-self-assessment-penalties-questions-and-responses.yml

```diff
@@ -1,30 +1,29 @@
 ---
 :which_year?:
-- 2011-12
```

Comment:
If it is mandatory that we update those dates, would it be possible to do on the initial commit, so we can see the differences in forthcoming commits?